pkg/azurefile/azurefile.go

```diff
@@ -928,21 +932,22 @@ func (d *Driver) copyFileShare(ctx context.Context, req *csi.CreateVolumeRequest
 	srcPath := fmt.Sprintf("https://%s.file.%s/%s%s", accountName, storageEndpointSuffix, srcFileShareName, accountSasToken)
 	dstPath := fmt.Sprintf("https://%s.file.%s/%s%s", accountName, storageEndpointSuffix, dstFileShareName, accountSasToken)
 
-	jobState, percent, err := getAzcopyJob(dstFileShareName)
+	aFunc := d.azcopyFunc
```

Comment:
why creating a new var instead of just using `d.azcopy`?

Reply:
Just use `d.azcopy` instead.